db/seeds.rb

```diff
@@ -8,10 +8,21 @@
   city: "Boston",
   state: "MA",
   zip: "01234",
+  entrance_fee: 1.00,
+  picture_url: "http://www.wow.com",
+  description: "This is the best beach.",
   name: "Tom Ford Beach",
   user_id: 1
 )
 
+Review.find_or_create_by!(
+  rating: 5,
+  description: "cool spot",
+  beach_id: 1,
+  user_id: 1,
+  score: 0
+)
+
 user = User.create!(
   email: Faker::Internet.email,
   password: "password",
```

Comment:
I would write the seed file in an order where you don't have to hardcode ids:

```
create users

create beaches based on above users

create reviews based on above users and beaches
```

something like this:

```ruby
user = User.create!(blah)
beach = Beach.find_or_create_by!(user_id: user.id, blah)
review = Beach.find_or_create_by!(user_id: user.id, beach_id: beach.id, blah)
```